app/containers/Incentives/IncentiveCalculatorContainer.tsx

```diff
@@ -13,6 +14,30 @@ export const IncentiveCalculator: React.FunctionComponent<Props> = ({
   applicationRevision
 }) => {
   const {edges = []} = applicationRevision.ciipIncentive;
+
+  const aggTotals = edges.reduce(
+    function calcAggs(
+      accumulator,
+      {node: {incentiveProduct, incentiveProductMax}}
+    ) {
+      return {
+        incentiveProduct:
+          accumulator.incentiveProduct + Number(incentiveProduct),
+        incentiveProductMax:
+          accumulator.incentiveProductMax + Number(incentiveProductMax)
+      };
+    },
+    {incentiveProduct: 0, incentiveProductMax: 0}
+  );
+
+  // incentive ratio = calculated incentive / maximum incentive
+  // set the default to 1 to avoid division by zero.
+  let aggIncentiveRatio = 1;
+  if (aggTotals.incentiveProductMax > 0) {
+    aggIncentiveRatio =
+      aggTotals.incentiveProduct / aggTotals.incentiveProductMax;
+  }
+
```

Comment:
Double-checked the math to make sure that this is equivalent to a weighted average. It is.